src/MergePlugin.php

```diff
@@ -422,6 +429,41 @@ protected function mergeLinks(array $origin, array $merge, array &$dups)
     }
 
     /**
+     * Merge the extras sections of two config files.
+     *
+     * @param RootPackage $root The root package.
+     * @param CompletePackage $package The imported package to merge.
+     */
+    public function mergeExtras(
+        RootPackage $root,
+        CompletePackage $package
+    ) {
+        $this->debug('Merging extra section');
+        $packageExtra = $package->getExtra();
+        if (empty($packageExtra)) {
+            return;
+        }
+
+        if (isset($packageExtra['merge-plugin'])) {
+            $this->debug("Skipping merge-plugin key");
+            unset($packageExtra['merge-plugin']);
+        }
+        $rootExtra = $root->getExtra();
+        foreach ($packageExtra as $key => $value) {
+            $this->debug("Merging extra key <comment>{$key}</comment>");
+            if (isset($rootExtra[$key])) {
+                $name = substr($package->getPrettyName(), 13);
+                throw new \OverflowException(
```

Comment:
Sure, np. As I've never seen this style of 'use' for SPL or other root namespace classes I'm curious to understand the thinking behind it.